js/downloads.js

```diff
@@ -22,7 +22,7 @@ var packagesV8 = [hadoop2p7, hadoop2p6, hadoopFree, sources];
 // 2.4.2+
 var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p11_hadoopFree, sources];
 
-addRelease("2.4.2", new Date("04/23/2019"), packagesV9, true);
+addRelease("2.4.3", new Date("05/07/2019"), packagesV9, true);
```

Comment:
According to the vote result announcement, `05/06` will be correct.
- https://lists.apache.org/thread.html/0efe232724f9696225d62dba0377f6ce39b6317b69493a3de0f9b7a3@%3Cdev.spark.apache.org%3E

Reply:
The vote passed on 05/06, but the release actually happened next day. See the release date of maven repo: https://mvnrepository.com/artifact/org.apache.spark

Reply:
Then, could you document this in the release manager's guide? This seems to be arguable. Previously, the vote result announce date is used.
cc @cloud-fan

Reply:
The release process usually takes days, so I pick the date that passed the vote. It's also fine to pick the date that started the release process.